# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderTextureHost.h
# Commit: 81601a02d9b7
# Full Hash: 81601a02d9b7e658f37f9525cce902eed94d678f
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-05 16:09:03
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Support using D3D11 textures as external compositor surfaces with D3D11 compositor. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95800
# ==============================================================================

diff -r 72d4bf5b093e -r 81601a02d9b7 gfx/webrender_bindings/RenderTextureHost.h
--- a/gfx/webrender_bindings/RenderTextureHost.h	Thu Nov 05 09:15:05 2020 +0000
+++ b/gfx/webrender_bindings/RenderTextureHost.h	Thu Nov 05 09:15:28 2020 +0000
@@ -25,6 +25,7 @@
 
 class RenderCompositor;
 class RenderDXGITextureHost;
+class RenderDXGIYCbCrTextureHost;
 class RenderMacIOSurfaceTextureHost;
 class RenderBufferTextureHost;
 class RenderTextureHostSWGL;
@@ -69,6 +70,9 @@
   virtual bool SyncObjectNeeded() { return false; }
 
   virtual RenderDXGITextureHost* AsRenderDXGITextureHost() { return nullptr; }
+  virtual RenderDXGIYCbCrTextureHost* AsRenderDXGIYCbCrTextureHost() {
+    return nullptr;
+  }
 
   virtual RenderMacIOSurfaceTextureHost* AsRenderMacIOSurfaceTextureHost() {
     return nullptr;